models.js

```diff
@@ -141,3 +141,13 @@ exports.fetchUsers = () => {
     })
   }
 
+
+  exports.eraseComment = (comment_id) => {
+    const query = `DELETE FROM comments WHERE comment_id = $1
+    RETURNING *;`
+    return db.query(query, [comment_id])
+    .then((comments) => {
+      console.log(comments.rows[0])
```

Comment:
can delete this console log, but you do want some way of checking that the delete was successful; a condition in this .then block will do